geocity/apps/accounts/users.py

```diff
@@ -70,15 +71,32 @@ def get_users_list_for_integrator_admin(user):
         )
     ]
 
-    return (
+    qs = (
         User.objects.annotate(
-            email_domain=Substr("email", StrIndex("email", Value("@")) + 1)
+            email_domain=Substr("email", StrIndex("email", Value("@")) + 1),
         )
         .filter(
-            Q(is_superuser=False),
-            Q(email_domain__in=email_domains) | Q(email__in=emails),
+            Q(is_superuser=False),  # par superuser
+            Q(email_domain__in=email_domains) | Q(email__in=emails),  # email autorise
```

Comment:
Remove thoses comment, and add logic of this filter ? It could be hard to read the order